Modules/Microsoft.PowerShell.UnixCompleters/Microsoft.PowerShell.UnixCompleters/BashUtilCompleter.cs

```diff
@@ -0,0 +1,313 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please use the current Copyright header
```suggestion
﻿// Copyright (c) Microsoft Corporation.
```